### ISSUE TITLE
API unable to have multiple requests going on at once?

### ISSUE THREAD

**Issue by daurnimator** [felipedaragon/remy#16]
Looking at the remy API it doesn't seem like it allows multiple requests to be handled at once?

**Comment by felipedaragon** [comment]
Hi @daurnimator, can you elaborate? AFAIK, there is no such limitation

**Comment by daurnimator** [comment]
You can't have rely handle two requests being handled in tandem: there is only one `emu.request` variable where state lives.

**Comment by felipedaragon** [comment]
There is only one request structure per request because this is how it is with modlua and Remy is simply emulating it: http://modlua.org/api/request

**Comment by daurnimator** [comment]
Yes there is only one request structure per request, but remy should allow more then one request per lua state.
